test/Compiler.test.js

```diff
@@ -269,11 +269,11 @@ describe("Compiler", () => {
 			mode: "production",
 			entry: "./missing",
 			output: {
-				path: "/",
+				path: "/directory",
 				filename: "bundle.js"
 			}
 		});
-		compiler.outputFileSystem = new MemoryFs();
+		compiler.outputFileSystem = new createFsFromVolume(new Volume());
```

Comment:
I guess the leading `new` should be omitted. `createFsFromVolume` sounds like a function and not a class.

Reply:
Good catch :clap: